.gitignore

```diff
@@ -48,3 +48,6 @@ spec/examples.txt
 config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml
+
+
+.DS_Store
```

Comment:
@marlo-longley could you add this to your local global gitignore instead? I think that's our preferred practice for IDE/OS files we want to ignore: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Thanks for the link. Removed. @corylown